app/register_validation.py

```diff
@@ -0,0 +1,140 @@
+import re
+
+from app.utilities.utilities import *
+from datetime import datetime
+
+
+def validate_password(password) -> bool:
+
+    # at least one capital letter
+    capitalLetter = r"[A-Z]"
+    # at least one small letter
+    smallLetter = r"[a-z]"
+    # at least one number
+    oneNumber = r"\d"
+    # min 8 characters, max 20
+    numberOfCharacters = r".{8,20}"
+
+
+    # boolean operation check it/cast bool
+    return bool(re.search(capitalLetter, password) and re.search(smallLetter, password) and re.search(oneNumber, password) and re.search(numberOfCharacters, password))
+    
+def validate_email(email):
+
+    email_pattern = r"((.*)@(.*))"
```

Comment:
that was good exercise too, but time to take some regex from web xD

Reply:
I will do this in the next PR

Reply:
solviing validation problem was big part of this PR solve it completely now imo

Reply:
each PR should introduce some fully implemented features, some things can wait for next PRs if they are side issues or big enough for separate PR, but validation is what you mostly worked on, so it should be finished completely. imo